src/System.Runtime/src/System.Runtime.builds

```diff
@@ -2,9 +2,15 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Runtime.CoreCLR.csproj">
+    <Project Include="System.Runtime.csproj">
       <AdditionalProperties>TargetGroup=</AdditionalProperties>
     </Project>
+    <Project Include="System.Runtime.csproj">
```

Comment:
@weshaggard Does that sound right? (That's the change I just made to my auto-generated builds files, so hopefully it's right :smile: )